pagefaultgames/pokerogue
# [QOL] Login now shows "beta" when playing on beta

## What are the changes the user will see?

On Beta it will show "Beta" next to "Login" on the login screen
See local PR here: <TODO for LUGIAD>

## Why am I making these changes?


Sangara asked me to. Probably to know easier if screenshots of login pages are from beta or not

## What are the changes from a developer perspective?

we create a new local key and all it based on the the server URL

## Screenshots/Videos

TBD (need locales still)

## How to test the changes?

???
- Play local, should not be there
- Should be there once its on beta. (you could try to change the code to your own server url to see if it appears i guess)

## Checklist

Are there any localization additions or changes? If so:
